src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+
+    {person.age && <p className="Person__age">I am {person.age}</p>}
+
+    <p className="Person__partner">
+      {person.isMarried === true
+        ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
+        : 'I am not married'}
```

Comment:
```suggestion
        ? `${partnerName} is my ${sex === MAN ? 'wife' : 'husband'}`
        : 'I am not married'}
```
Consider creating `const MAN = 'm'`